beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestIntegrationInvoiceWithRepairLogic.java

```diff
@@ -1448,4 +1449,69 @@ public void testRegressionFor170() throws Exception {
         checkNoMoreInvoiceToGenerate(account);
 
     }
+
+    @Test(groups = "slow")
+    public void testWithAdjustementsAndCancellation() throws Exception {
```

Comment:
Nit: typo (adjustments).

Reply:
Fixed in https://github.com/killbill/killbill/pull/1731/commits/cc62b5e80cbe737e949e01fe8d1d8e389982473e